pkg/datapath/linux/probes/probes.go

```diff
@@ -487,3 +434,171 @@ func HaveV3ISA() error {
 	}
 	return nil
 }
+
+// CreateHeaderFiles creates C header files with macros indicating which BPF
+// features are available in the kernel.
+func CreateHeaderFiles(headerDir string, probes *HeaderProbes) error {
+	common, err := os.Create(filepath.Join(headerDir, "features.h"))
+	if err != nil {
+		return fmt.Errorf("could not create common features headerfile: %w", err)
+	}
+	defer common.Close()
+	err = writeCommonHeader(common, probes)
```

Comment:
Here and in the linew below:

```suggestion
	if err := writeCommonHeader(common, probes); err != nil {
```